README_Docker_and_RA.md

```diff
@@ -81,8 +81,8 @@ Or, you can [build and install][sgx-pck-id-retrieval-tool] it yourself.
 
 2. Retrieve your machine's FMSPC by running the following command:
 
```

Comment:
@lagunovsky The old version seems to work better, could you revert?

Reply:
@RogerLamTd sure, done